### PR TITLE
Create pull request template

### LINKED ISSUE #471: Pull request template
Based on @Darhazer’s first comment in #470, I’m wondering if this repository shouldn’t have a pull request template?

See https://github.com/blog/2111-issue-and-pull-request-templates for details.

### PR DESCRIPTION
**Pull request by Darhazer**
* Used the rubocop's ones as templates.
* Added more instructions for contribution, including some pitfalls
* Fixes #471

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* Be clear, concise and precise in your description of the problem.
+* Open an issue with a descriptive title and a summary in grammatically correct,
+  complete sentences.
+* Report the versions of `rubocop-rspec`, as well as the output of `rubocop -V`
+* Include any relevant code to the issue summary.
+
+## Pull requests
+1. Fork the project.
+2. Create a feature branch.
+3. Commit your changes.
```
Comment (bquorning) [inline .github/CONTRIBUTING.md]: I’d say that “commit your changes” should come *after* “add tests” and “add changelog entry”.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+4. Make sure to add tests.
+5. Make sure the test suite is passing (run `rake`).
+6. Add [Changelog](../blob/master/CHANGELOG.md) entry.
+7. Push to the branch.
+8. Create new Pull Request.
+
+## Creating new cops
+There are some steps you have to follow when you add new cops:
+* Add an entry to `config/default.yml`. It's ordered list, make sure to follow the order.
+* The description of the cop in the code should match the one in config. `bin/build_config` takes care of this.
```
Comment (bquorning) [inline .github/CONTRIBUTING.md]: It’s not clear from this description how `bin/build_config` takes care of this. Will the script copy the description over so I don’t need to do it manually, or does it just verify that they are identical?

**Comment by Darhazer** [comment]
I have resolved @bquorning comments and I'm going to merge this, as we have more contributors recently 
 :)
It can be improved further if needed with another PR

**Comment by bquorning** [comment]
:+1:

**Comment by dgollahon** [comment]
This looks solid guys--sorry I didn't come by and review it earlier. Good call on merging as is, we can always tweak it later.

**Pull request merged**